microsoft-edge/devtools-guide-chromium/whats-new/2021/10/devtools.md

```diff
@@ -0,0 +1,98 @@
+
+<!-- Title: CSS Mirror Editing in Visual Studio Code -->
+<!-- Subtitle: Changes in the CSS Styles editor now show up in your CSS files inside Visual Studio Code. -->
+
+The Microsoft Edge DevTools extension for Visual Studio Code now features an experiment called **CSS Mirror Editing**.  Using this feature, you can use the **Styles** tool to adjust your CSS, and the changes you apply are automatically added to the CSS source files in the Visual Studio Code editor.  You can toggle this functionality on and off.
+
+For more information, see [Syncing live changes from the Styles tool by using CSS Mirror Editing](../../../../visual-studio-code/microsoft-edge-devtools-extension.md#syncing-live-changes-from-the-styles-tool-by-using-css-mirror-editing).
+
+To provide feedback, in Visual Studio Code, in the **Activity Bar**, click **Microsoft Edge Tools**, and then in the **CSS Mirror Editing** section, click the **leaving feedback here** link.
+
+:::image type="content" source="../../media/2021/10/css-mirror-editing-button.msft.png" alt-text="Changes in the CSS Styles editor now show up in your CSS files inside Visual Studio Code." lightbox="../../media/2021/10/css-mirror-editing-button.msft.png":::
+
+
+<!-- ====================================================================== -->
+## All error and warning messages in the Console now have a Search Web icon
+
+<!-- Title: All console errors and warnings now have a Search Web icon -->
+<!-- Subtitle: You can now search for any of your console errors and warnings right from DevTools. -->
+
+The **Search for this message on the Web** icon is now available on all errors and warnings in the **Console**.  Previously, this icon only appeared on some of the common errors and warnings.  Now the icon has been added on the remaining errors and warnings.  The relevant error or warning string is used as the search string.
+
+For more information, see [Search for Console errors on the web](../09/devtools.md#search-for-console-errors-on-the-web).
+
+:::image type="content" source="../../media/2021/10/console-message-search-web-button.png" alt-text="All error and warning messages in the Console now have a Search Web icon." lightbox="../../media/2021/10/console-message-search-web-button.png":::
+
+
+<!-- ====================================================================== -->
+## Improved keyboard access for defining User-Agent Client Hints
+
+<!-- Title: Improved keyboard access when navigating to User agent client hints in Settings -->
+<!-- Subtitle: When adding a custom device to emulate in DevTools, you can now expand the User agent client hints section more easily. -->
+
+Starting with [Microsoft Edge version 92](../05/devtools.md#user-agent-client-hints-for-devices-in-the-network-conditions-tab), you can specify User-Agent Client Hints when defining the user agent string in the **Network conditions** tool or when adding a custom device to emulate in **Settings**.
```

Comment:
```suggestion
Starting with [Microsoft Edge version 92](../05/devtools.md#user-agent-client-hints-for-devices-in-the-network-conditions-tab), you can specify User-Agent Client Hints when [defining the user agent string in the **Network conditions** tool](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/device-mode/override-user-agent) or when [adding a custom device to emulate in **Settings**](https://docs.microsoft.com/microsoft-edge/devtools-guide-chromium/device-mode/#add-a-custom-mobile-device).
```
Again, I know we don't like in-line links but this feels like an opportunity to drive readers further into our docs to learn how to do either UA string overrides or set a new device to emulate